src/commands/trace/__tests__/trace.test.ts

```diff
@@ -36,5 +36,33 @@ describe('trace', () => {
         'circleci',
       ])
     })
+
+    test('should correctly detect the jenkins environment', () => {
+      process.env = {
+        DD_CUSTOM_TRACE_ID: 'abc',
+        JENKINS_HOME: '/root',
+        NON_JENKINS_ENV: 'bar',
+        WORKSPACE: 'def',
+      }
+      const command = new TraceCommand()
+      expect(command['getCIEnvVars']()).toEqual([
+        {
+          DD_CUSTOM_TRACE_ID: 'abc',
+          WORKSPACE: 'def',
+        },
+        'jenkins',
+      ])
+    })
+
+    test('should not detect the jenkins environment if it is not instrumented', () => {
+      process.env = {
+        // DD_CUSTOM_TRACE_ID: 'abc',
```

Comment:
This comment does not seem to add much here, could it be removed or if it is to show the environment variable is not defined, could it be made explicit that this env var is not supposed to be defined?